### ISSUE TITLE
Separation between tabs - each tabs have own status bar.

### ISSUE THREAD

**Issue by JoxCoswit** [tablacus/TablacusExplorer#502]
Gaku, 

Can status bar be seperted for each tabs? 
Is there any add-on that can solved it or could this be done through the settings/layout?
Honestly, unique style is good... but sometimes is little confusing-at least for me. 🤔
I look at the each tabs belongs as an individual workspace.

**_Both tabs share status bar among themself_**
![Unique](https://user-images.githubusercontent.com/60333254/145464995-f67817a5-66a5-4c5e-a8f9-be4420ac7709.jpg)

**_Each tabs have status bar for themself._**
![Seperted](https://user-images.githubusercontent.com/60333254/145465014-420f2fee-c323-40b6-877c-297161469a34.jpg)

Best regards,

**Comment by tablacus** [comment]
Please try to use an add-on "User style sheet".

```css
#BottomBar3Left, #BottomBar3Center {
	border-right: 1px solid #888;
}
```
![image](https://user-images.githubusercontent.com/5156977/145485836-5149e505-823f-4480-8816-861ecc2d7901.png)

![image](https://user-images.githubusercontent.com/5156977/145485915-e60f8c38-eb89-4dbd-be8f-686d9742ff97.png)

Regards,

**Comment by tablacus** [comment]
I have added "Internal Bottom bar" on Tablacus Explorer 21.12.10.

![image](https://user-images.githubusercontent.com/5156977/145587942-7e497768-8300-43bc-a266-6ad9e10bd218.png)

https://github.com/tablacus/TablacusExplorer/releases/tag/21.12.10

Regards,

**Comment by JoxCoswit** [comment]
I would ask for a little help, it is about configuration....
I'm trying to get status bar next to the recycle bin
This information on the status bar is important for me because I can quick see all the necessary info about files, folders, how many is selected etc.

I noticed I can't arrange it to be next to each other.
Also I can't figure out how to do that now to look just like before?

**I try to get this kind of look:** 
_all in one line_
![Example122](https://user-images.githubusercontent.com/60333254/145623425-f0d2c3ea-f820-4ece-a6bc-7d3c2f5b63fa.jpg)

**But closely I got is that - in two line**
_I would like somehow to get in one line (picture upper)_
![Example12](https://user-images.githubusercontent.com/60333254/145623667-b783b6cd-8101-4593-ac48-9f2f785c0bec.jpg)

**When I open the Status Bar settings field is gray**
_It is disabled and can't set it up next to each other_
![Status bar-conf position-2](https://user-images.githubusercontent.com/60333254/145624430-83cff7f9-a12a-4679-a580-dfc2dce04669.jpg)

**Example from last version:** 
Delete + Status Bar (next to each other)
![Status bar-conf position-3](https://user-images.githubusercontent.com/60333254/145624499-beada123-5d7c-4176-a350-0200042d6b45.jpg)

**Comment by JoxCoswit** [comment]
Gaku, 
I was thinking if you have a trouble to solve, and resolve, all that, and if you must adjusting everything else to adjust to work.

Maybe it's better to go back to the old version before these changes? 
You decide, how you want it - will you keep that changes or revert and go back to the old version?

Thank you for your effort and I appreciate that a lot but I wouldn’t want you to have to change and adjust all that. 

No waste your time anymore, please.

I’ll use it this way, I thought there would be less work and much easier.
Of course it would be nice to have something and like that - that’s why I suggested it. 

I thought it would be much easier to that...  
As far as I can see, you have to adjust everything.. TE, Add-ons.. status bar, icons. .etc. - who knows what else!

You helped me already by enabling functionality for deleting and that is really great feature, I have been using it all day.

That thing like "Separation tabs, and all about it" ...let that wait for some other time.  I hope you agree with me? 

Best regards,

**Comment by JoxCoswit** [comment]
**Quick update:** 

I managed to get a bigger recycle bin icon by setting a slightly higher numeric value on the card "Icon"
It seems to me that I did it yesterday but I didn’t set enough higher value to look as before, today I increased more. 
'By default' icons in "Internal Bottom bar" are almost twice smaller than in classic "Bottom bar"-or it just look so to me.

Best regards.

**Comment by tablacus** [comment]
1. I have also added Internal bar support for the status bar.
2. The default icon size for the Internal bar is 16; the default icon size for the Tool bar is 24.
You can change the size of the icon.
Tools -> Add-ons -> Delete -> Icon -> Size
![image](https://user-images.githubusercontent.com/5156977/145679764-5c8a7460-ed2c-4826-bbd8-d8146799b276.png)

![image](https://user-images.githubusercontent.com/5156977/145679828-b7759b88-878e-4e9b-b7da-863827dbad51.png)

Regards,

**Comment by JoxCoswit** [comment]
Gaku, 

Thanks for the advice, for default icon size toolbar. As, I wrote earlier today - I tried to set up a little different values. 
"By the eye" I set up on 30 (while yesterday I put was at 15-20)
Because icon 'Recycle Bin' in "Internal Bottom bar" is reduced at least for half than in the classic "Bottom bar".

P.S. I made a slightly different arrangement of icons. I placed the "Recycle Bin" at the end in front of the "Size Bar".

**That would be all, I can only confirm everything is working properly today.... thanks again**! 😃 

_It look like this now:_
![EXAMPLE2](https://user-images.githubusercontent.com/60333254/145682444-6aabe9da-e107-485a-b506-71d9ac071381.jpg)
